chain/x/zoracle/client/rest/query.go

```diff
@@ -25,3 +27,36 @@ func getRequestHandler(cliCtx context.CLIContext, storeName string) http.Handler
 		rest.PostProcessResponse(w, cliCtx, res)
 	}
 }
+
+func getScriptHandler(cliCtx context.CLIContext, storeName string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		codeHash := vars[codeHash]
```

Comment:
This will never fail, right?

Reply:
How to fail?

Reply:
There's a bit of variable shadowing in here, right? the right codeHash is the global variable, while the left is a new variable. Is this intended?

What happens if user does not specify codeHash? I guess it will try to query with empty string and fail on the line after this, correct?

Reply:
1. Good catch change name already
2. If not specify, route will be incorrect and not execute handler function (404 return)